tests/Microsoft.Identity.Test.Unit/AppConfigTests/ConfidentialClientApplicationBuilderTests.cs

```diff
@@ -41,8 +41,6 @@ public void TestConstructor()
             // Validate Defaults
             Assert.AreEqual(LogLevel.Info, cca.AppConfig.LogLevel);
             Assert.AreEqual(TestConstants.ClientId, cca.AppConfig.ClientId);
-            Assert.IsNotNull(cca.AppConfig.ClientName);
```

Comment:
What are they if not null?

Reply:
Since we are changing this to accept the api id of the caller sdk, these can be null or empty when not passed in by the user. I can add the check to validate that these are null by default. And have a value when a value is passed in